### PR TITLE
Unique IDs

### PR DESCRIPTION
**Pull request by mwasilew**

### CODE REVIEW

File: source/interfaces.rst [hunk 1]
```diff
@@ -56,6 +56,9 @@ For some i.MX - for example - only a few serial ports and pin-mux settings can
 be used to run the vendor's RAM-calibration tool. Ask your software developer
 for specific restrictions for your CPU.
 
+If UART connectivity is done using UBS device, it should have unique ID
```
Comment (SmithChart) [inline source/interfaces.rst]: Could you correct this typo?

File: source/interfaces.rst [hunk 2]
```diff
@@ -89,7 +92,9 @@ non-volatile memory.
 
 On some CPUs the download can be done via USB. USB is convenient because it
 features a relatively high bandwidth and does not need expensive external
-components to be used.
+components to be used. Each board should present unique ID to the host. If IDs
+are identical for all boards there is no way to drive such devices in
+fully automated environment.
```
Comment (SmithChart) [inline source/interfaces.rst]: I disagree. It is still possible to identify a device based on it's USB-path. I would prefer a _softer_ statement here.

**Review by SmithChart — CHANGES_REQUESTED** [review]
Please have a look at my comments.

File: source/interfaces.rst (hunk 2 above)
Comment (mwasilew) [reply source/interfaces.rst]: I tried and it didn't end well so I'll won't change my mind. But in principal you're right. I'll update wording.

**Review by SmithChart — APPROVED** [review]
Thanks for your changes.

**Pull request merged**